react/components/Tag/index.js

```diff
@@ -22,30 +22,49 @@ class Tag extends PureComponent {
   }
 
   render() {
-    const { children, onClick, disabled, color, bgColor } = this.props
+    const {
+      children,
+      onClick,
+      disabled,
+      color,
+      bgColor,
+      type,
+      variation,
+    } = this.props
 
     const baseClasses = 'br-pill t-small pv2 ph4 dib fw5'
 
     let theme = ''
-    switch (this.props.type) {
+    const variationIsLow = variation === 'low'
+
+    switch (type) {
```

Comment:
You could replace this for a get function here and call it directly in `className`, What you think?

Reply:
Could you show me another component with an example of that?
What are the benefits?